contrib/src/test/java/com/datatorrent/contrib/parser/CsvPOJOParserTest.java

```diff
@@ -319,6 +331,19 @@ public void TestParserMoreFields()
   }
 
   @Test
+  public void TestParserMoreFieldsOnlyPOJOPortConnected()
+  {
+    parser.parsedOutput.setSink(null);
+    parser.beginWindow(0);
+    parser.in.process(
+        "1234,98233,adxyz,0.2,2015-03-08 03:37:12,11/12/2012,12,y,OPTIMIZATION,CAMP_AD,Y,yes,ExtraField".getBytes());
+    parser.endWindow();
+    Assert.assertEquals(0, objectPort.collectedTuples.size());
+    Assert.assertEquals(0, pojoPort.collectedTuples.size());
+    Assert.assertEquals(1, error.collectedTuples.size());
```

Comment:
Can you add small addition to make sure that error port has same string tuple that was passed on input port?